tests/src/test/fdc3.raiseIntent.ts

```diff
@@ -1,32 +1,150 @@
 import { ResolveError } from "@finos/fdc3";
-
-const ExpectedErrorNotThrownError = "Expected error NoAppsFound not thrown";
+import { assert, expect } from "chai";
 
 export default () =>
-  describe("fdc3.raiseIntent", async () => {
-    it("Method is callable", async () => {
-      const context = {
-        type: "ThisContextDoesNotExist",
-        name: "Name",
-        id: {
-          ticker: "ticker",
-          ISIN: "US0378331005",
-          CUSIP: "037833100",
-          FIGI: "BBG000B9XRY4",
+  describe("fdc3.raiseIntent", () => {
+    before(async () => {
+      await window.fdc3.getOrCreateChannel("fdc3.raiseIntent");
+      await window.fdc3.joinChannel("fdc3.raiseIntent");
+    });
+
+    beforeEach(async () => {
+      await window.fdc3.broadcast({ type: "closeWindow" });
+      await new Promise<void>((resolve) => setTimeout(() => resolve(), 1000)); // Wait until close window event is handled
+    });
+
+    it("Should start app B when raising intent 'sharedTestingIntent1' with context 'testContextY'", async () => {
+      const intentResolution = (await window.fdc3.raiseIntent(
+        "sharedTestingIntent1",
+        {
+          type: "testContextY",
+        }
+      )) as any;
+
+      expect(intentResolution.source).to.eq("IntentAppBId");
+      expect(intentResolution.intent).to.eq("sharedTestingIntent1");
+    });
+
+    it("Should start app A when targeted by raising intent 'aTestingIntent' with context 'testContextX'", async () => {
+      const intentResolution = (await window.fdc3.raiseIntent(
```

Comment:
I've noticed a few places where you are casting to `any`. This is because the API we are currently using is v1.2, where, for example IntentResolution has the following API:

https://fdc3.finos.org/docs/1.2/api/ref/Metadata#intentresolution

Whereas in v2.0 it is as follows:

https://fdc3.finos.org/docs/api/ref/Metadata#intentresolution

Note, the `intent` property is a v2.0 feature.

For now, I'd recommend implementing v1.2 tests, we can raise an issue for the 'upgrade' tasks as a next iteration.

Reply:
Yep this was to get around keeping the version at v1.2. I will remove the tests that are like this for now